res/css/views/rooms/_EventTile.pcss

```diff
@@ -822,6 +822,9 @@ $left-gutter: 64px;
     }
 }
 
+.mx_EventTile_e2eIcon_warning {
+    align-self: center;
```

Comment:
i dont think this is the behgaviour we want, the shield should be aligned with the first line of text, but not aligned with the center when a block goes over multiple lines

Reply:
Oh ok. I will fix this.